components/core/src/clp_s/search/ColumnScan.cpp

```
@@ -0,0 +1,537 @@
+            bitmap[message_index] |= compare(operation, value, operand) ? 1 : 0;
+        }
+    }
+    return bitmap;
+}
+
+[[nodiscard]] auto
+clp_string_matches(ClpStringColumnReader* reader, clp::Query const& query, uint64_t message_index)
+        -> bool {
+    auto const value = std::get<std::string>(reader->extract_value(message_index));
+    auto const matches_wildcard = [&query, &value]() -> bool {
+        return clp::string_utils::wildcard_match_unsafe(
```

Cppcheck (2.20.0)

[style] The function 'descriptor_end' is never used.

(unusedFunction)